app/models/choice.rb

```diff
@@ -2,6 +2,8 @@ class Choice < ActiveRecord::Base
   belongs_to(:answer, :inverse_of => :choices, :touch => true)
   belongs_to(:option, :inverse_of => :choices)
 
+  validates :answer, :option, presence: true
```

Comment:
I am not a believer in testing the builtin functionality of Rails. Thanks for asking though.

I also don't think we need an app-level validation here if we have null false in the DB. Do you?